roles/core.role/defaults/main.yml

```diff
@@ -1,2 +1,27 @@
 ---
 # defaults file for core
+
+# System-wide hostname
+system_hostname: "ec2-xxxx-prod-vml-01.example.com"
```

Comment:
I added a section about this in the README and removed the hostname default. It should be set in the inventory, per host.